### ISSUE TITLE
Avatar disappears when window width is very small

### ISSUE THREAD

**Issue by naefun** [benawad/dogehouse#233]
Avatar profile picture shrinks to the point it disappears when reducing screen width very small.

Could be fixed by adding some flex-wrap and gap and remove some margin :)

**Comment by naefun** [comment]
Before --
![image_2021-02-15_155320](https://user-images.githubusercontent.com/43324580/107968169-ecea5800-6fa5-11eb-82d6-c31c9633791a.png)

After--
![image](https://user-images.githubusercontent.com/43324580/107968804-ba8d2a80-6fa6-11eb-9b41-19a9d153cc4a.png)
![image](https://user-images.githubusercontent.com/43324580/107968841-c547bf80-6fa6-11eb-9379-8982dcec721e.png)

**Comment by SuperSonicHub1** [comment]
@naefun Could you share the CSS you used? It's hard to see in your screenshots.

**Comment by SuperSonicHub1** [comment]
![Screenshot](https://user-images.githubusercontent.com/25624882/111017565-5e89ac00-8382-11eb-85c8-aa5f0393cec1.png)
@naefun I was able to replicate your work! I changed the `gap` from `18px` to `1rem` so that the icons were more evenly spaced out. To make sure that the buttons are vertically aligned when stacked on top of one another, I removed the margin from the `<div>`s wrapping around the buttons and made the padding of `ProfileButton` a prop so that it can be disabled specifically for the home screen; it's `true` by default as to not break the three other pages where it is used. If it looks good, I'd be happy to make a pull request.
